dotCMS/src/main/java/com/dotmarketing/osgi/OSGIProxyServlet.java

```diff
@@ -26,6 +19,7 @@
     public static DispatcherTracker tracker;
     public static ServletConfig servletConfig;
     public static BundleContext bundleContext;
+    private Boolean isInitialized = Boolean.FALSE;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this misleading mutable servlet instance field or make it "static" and/or "final" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2226)